src/services/DiscourseService.ts

```diff
@@ -0,0 +1,62 @@
+import logger from 'decentraland-gatsby/dist/entities/Development/logger'
+import { Avatar } from 'decentraland-gatsby/dist/utils/api/Catalyst'
+
+import { Discourse, DiscoursePost } from '../clients/Discourse'
+import { DISCOURSE_AUTH, DISCOURSE_CATEGORY } from '../entities/Discourse/utils'
+import { inBackground } from '../entities/Proposal/routes'
+import * as templates from '../entities/Proposal/templates'
+import { forumUrl, proposalUrl } from '../entities/Proposal/utils'
+
+import { ProposalInCreation } from './ProposalService'
+import { SnapshotService } from './SnapshotService'
+
+export class DiscourseService {
+  static async createProposal(
+    data: ProposalInCreation,
+    proposalId: string,
+    profile: Avatar | null,
+    snapshot_url: string,
+    snapshotId: string
+  ) {
+    let discourseProposal: DiscoursePost
+    try {
+      const discourseTemplateProps: templates.ForumTemplateProps = {
+        type: data.type,
+        configuration: data.configuration,
+        user: data.user,
+        profile,
+        proposal_url: proposalUrl({ id: proposalId }),
+        snapshot_url,
+        snapshot_id: snapshotId,
+      }
+
+      discourseProposal = await Discourse.get().createPost(
+        {
+          category: DISCOURSE_CATEGORY ? Number(DISCOURSE_CATEGORY) : undefined,
+          title: templates.forumTitle(discourseTemplateProps),
+          raw: await templates.forumDescription(discourseTemplateProps),
+        },
+        DISCOURSE_AUTH
```

Comment:
Can we move `DISCOURSE_AUTH` to the DiscourseClient? 

Or if we want to think it as an external module maybe we can have a DiscourseClient initializer that returns the DiscourseClient.
E.g:
```
const DiscourseClient = new Discourse(DISCOURSE_AUTH)
...
DiscourseClient.createPost(newPost)
```